### ISSUE TITLE
The ? menu is an accessibility issue. Change to &Help

### ISSUE THREAD

**Issue by DReynolds4001** [DReynolds4001/SftwrEngMissionControl#3]
The ? menu should be renamed to &Help so it's accessibility friendly. There's literally no upside to calling this menu ?

Help is the one thing you want people to be able to find without thinking.

(Please note I've raised this before and it was closed, IMO, incorrectly. I'm not raising this issue because I have accessibility issues. I'm raising it because I care about those that do. #4601)

Link to original issue:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5632

**Comment by DReynolds4001** [comment]
Solution Implemented.
